lib/term.js

```diff
@@ -1,42 +1,55 @@
 const rdf = require('@rdfjs/data-model')
 
-function term (value, type, datatype) {
-  if (datatype) {
-    datatype = rdf.namedNode(datatype.value || datatype.toString())
+function blankNode (value) {
+  if (value && typeof value !== 'string') {
+    throw new Error('unknown type')
   }
 
-  if (type) {
-    if (type === 'BlankNode') {
-      return rdf.blankNode(value && value.toString())
-    }
+  return rdf.blankNode(value && value.toString())
```

Comment:
Isn't `value.toString()` unnecessary if you test the `typeof value`?
```suggestion
  return rdf.blankNode(value)
```